ee/watchdog/watchdog_task_windows.go

```diff
@@ -0,0 +1,140 @@
+//go:build windows
+// +build windows
+
+package watchdog
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log/slog"
+
+	"github.com/kolide/kit/version"
+	agentsqlite "github.com/kolide/launcher/ee/agent/storage/sqlite"
+	"github.com/kolide/launcher/pkg/launcher"
+	"github.com/kolide/launcher/pkg/log/multislogger"
+	"github.com/peterbourgon/ff/v3"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+// RunWatchdogTask is typically run as a check to determine the health of launcher and restart if required.
+// it is installed as an exec action via windows scheduled task. e.g. C:\path\to\launcher.exe watchdog -config <path>.
+// you can alternatively run this subcommand to install or remove the scheduled task via the --install-task or --remove-task flags
+func RunWatchdogTask(systemSlogger *multislogger.MultiSlogger, args []string) error {
```

Comment:
note that this file was renamed and looks new but most of what has changed here is just the options parsing logic. the rest is a much simpler version (no power event watching, no polling) of what the service version did before